bundles/org.openhab.binding.teslapowerwall/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="teslapowerwall"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="tesla-powerwall">
+		<label>Tesla Powerwall</label>
+		<description>Tesla Powerwall</description>
+
+		<channels>
+			<channel id="gridstatus" typeId="tesla-powerwall-gridstatus"/>
```

Comment:
```suggestion
			<channel id="gridstatus" typeId="grid-status"/>
```
All type id's have the `tesla-powerwall` prefix, that is not needed as they already belong the the `tesla-powerwall` binding and are unique in that way. 

Can you also check if you can re-use some types as there are some very similar types with only a different label.

Reply:
Done, except  I don't quite understand the " re-use some types as there are some very similar types with only a different label" comment.